xml/admin_monitoring_alerting.xml

```diff
@@ -1231,7 +1227,7 @@ monitoring:
             <itemizedlist>
               <listitem>
                 <para>
-                  <literal>full</literal> - the cluster is flagged as full and cannot serve writes
+                  <literal>full</literal> - the cluster is flagged as <literal>full</literal> and cannot serve writes
```

Comment:
I don't think we should use standalone hyphens like this. Either an em-dash or possibly a full colon, and this should be consistent down the list.

Reply:
What's our style guide? I have no idea, actually. Colon seems to be the go.

Reply:
i sometimes use normal hyphens, but....in this case, i would use &lt;variablelist> for explaining terms or flags